packages/js/client/src/__tests__/utils/validate/wrapper-a/package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Could the new test wrappers be moved to `test-cases`? Or is it better if they are here?

Reply:
I will leave them here. The reason is that there's no folder `client` specific there, since we want all the clients to have their own repo, I would suggest having this here